smartystreets_python_sdk/us_street/client.py

```diff
@@ -14,8 +14,6 @@ def send_lookup(self, lookup):
         """
         Sends a Lookup object to the US Street API and stores the result in the Lookup's result field.
         """
-        if not lookup.street:
-            return
```

Comment:
What's the purpose of this change?

Reply:
That change was reverted 5 hours after it was made. It's foggy the reason that we executed it.

Reply:
I'm afraid I don't follow. What does removing the `if` statement have to do with adding support for the `license` query string parameter?